include/itkRieszRotationMatrix.hxx

```diff
@@ -209,7 +208,7 @@ RieszRotationMatrix< T, VImageDimension >
   InternalMatrixType & S = this->GetVnlMatrix();
   if ( this->m_Order == 1 )
     {
-    S = this->GetSpatialRotationMatrix().GetVnlMatrix();
+    S = this->GetSpatialRotationMatrix().GetVnlMatrix().as_matrix();
```

Comment:
What's the difference, why is `as_matrix()` needed?

Reply:
This is for type matching.  GetVnlMatrix() returns a vnl_fixed_matrix<r,c>, but InternalMatrixType is vnl_matrix.  The old implicit (and somewhat dangerous) conversions are now disabled and require explicit matching.